ui/jobs/jobs.js

```diff
@@ -28,7 +29,7 @@ var Options = {
   BigBuffBorderSize: 1,
 
   FarThresholdOffence: 24,
-
+  LowMPThreshold: 4800,
```

Comment:
This option seems very job specific, so probably should have Drk in its name?

Reply:
I had thought to use it elsewhere, BLM for example, but that would have different values anyway. So I agree, just don't know what to change it to.

Reply:
I think just sticking a "Drk" at the front would work.